app/Http/Controllers/VueController.php

```diff
@@ -52,4 +59,32 @@ public function view(?string $albumId = null, ?string $photoId = null): View
 
 		return view('vueapp');
 	}
+
+	/**
+	 * Check if user can access the album.
+	 *
+	 * @param AbstractAlbum $album
+	 *
+	 * @return bool true if access, false if password required
+	 *
+	 * @throws UnauthorizedException if user is not authorized at all
+	 */
+	private function check(AbstractAlbum $album): bool
+	{
+		$result = Gate::check(AlbumPolicy::CAN_ACCESS, [AbstractAlbum::class, $album]);
+
+		// In case of a password protected album, we must throw an exception
```

Comment:
This comment isn't correct anymore, is it?

Reply:
[092ce89](https://github.com/LycheeOrg/Lychee/pull/2688/commits/092ce890d58e54d4f86dba2327f63e9356b066ed) fixed. :) Good spot.